googler

```diff
@@ -2291,6 +2298,10 @@ class Result(object):
                 print(' ' * (indent + pre) + metadata)
 
         if colors:
+            # Start from the last match, as inserting the bold characters changes the offsets.
+            matches.reverse()
+            for match in matches:
+                abstract = abstract[:match['offset']] + '\033[1m' + match['phrase'] + '\033[0m' + abstract[match['offset']+len(match['phrase']):]
```

Comment:
Line too long, break it up; also, for consistency, use spaces around operators. Below is black's take on this statement, with quotes replaced. (I'm not a fan of black, and it's obviously too late to blacken googler, but it does stop all arguments on code formatting.)

```py
abstract = (
    abstract[: match['offset']]
    + '\033[1m'
    + match['phrase']
    + '\033[0m'
    + abstract[match['offset'] + len(match['phrase']) :]
)
```